src/Features/Core/Portable/ExtractMethod/MethodExtractor.Analyzer.cs

```diff
@@ -342,27 +339,6 @@ private Dictionary<ISymbol, List<SyntaxToken>> GetSymbolMap()
                 return symbolMap;
             }
 
-            private DataFlowAnalysis GetDataFlowAnalysisData()
-            {
-                if (SelectionResult.IsExtractMethodOnExpression)
-                    return this.SemanticModel.AnalyzeDataFlow(SelectionResult.GetNodeForDataFlowAnalysis());
-
-                var (firstStatement, lastStatement) = this.SelectionResult.GetFlowAnalysisNodeRange();
-                return this.SemanticModel.AnalyzeDataFlow(firstStatement, lastStatement);
-            }
```

Comment:
a high level goal was to stop exposing the 'start/end nodes' and having callers then do things like AnalyzeDataFlow/AnalyzeControlFlow.  By the end of hte cleanup, the nodes are not exposed, and that data is computed *once* and cached for all callers to use.